src/server/pps/cmds/cmds.go

```diff
@@ -698,6 +654,76 @@ All jobs created by a pipeline will create commits in the pipeline's repo.
 	return result, nil
 }
 
+// ByCreationTime is an implementation of sort.Interface which
```

Comment:
Yup, I generally like to have the public stuff at the top of the file so it's what you start reading.